lib/vsc/filesystem/lustre.py

```diff
@@ -103,19 +102,19 @@ def __init__(self, mountpoint):
 
         project_locations = ['gent', 'gent/vo/00[0-9]']
         projectid_maps = {'gvo' : 900000}
-        super(LustreVscGhentScratchFs, self).__init__(mountpoint, project_locations, projectid_maps)
+        super().__init__(mountpoint, project_locations, projectid_maps)
 
 class LustreVscScratchFs(LustreVscFS):
     def __init__(self, mountpoint):
         project_locations = [os.path.join(SCRATCH_SUBDIR, PROJECTS_SUBDIR), os.path.join(SCRATCH_SUBDIR, USERS_SUBDIR)]
         projectid_maps = {}
-        super(LustreVscScratchFs, self).__init__(mountpoint, project_locations,
+        super().__init__(mountpoint, project_locations,
             projectid_maps)
 
     def pjid_from_name(self, name):
         del name
         self.log.error('Use explicit mapping')
-        raise Exception('Can not use pjid_from_name')
+        raise ValueError(f'Can not use pjid_from_name')
```

Comment:
why?

Reply:
an Exception is considered too broad, and you should always give a bit of a hint of what is going wrong. e.g. raise an actual error subclass from Exception. it's not enforced, but it is deemed good programming practice.